### DISCUSSION TITLE
bluetooth serial

### DISCUSSION THREAD

**Discussion by noweare1** [Mair/esp32-course#126] [Course Qeustions]
First time I went through the course I did not have a HC-05 bt module. I have one now and having trouble getting it to log to vsc terminal. I am using ubuntu and can pair and connect the hc-05 to BT controller in my linux desktop. Then I connect rfcomms to it successfully. Then from VSC I run the command  idf.py -p /dev/rfcomm0 monitor, This is after downloading the code to the esp32 the usual way. 
I do not  see anything coming over bluetooth to the monitor as it this point be using rfcomm to capture the data and output to the monitor.  Anyone able to do this lesson using ubuntu and hc-05 ?

Thanks in advance

**Comment by Mair** [comment]
@noweare1 
If you short RX and TX and then send data to it via a program that can send serial data. You should see it echo what you type. Give that a try and lets see how that goes.
